tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>"low-bit clusers." -&gt; clusters. And low-bit is a new way of referencing this. for fewer clusters? Or is this talking about post-training quantization.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>If low-bit is confusing, we can use fewer clusters. </p><p>Given k clusters, we need log_2(k) bits to encode the index of each cluster in clustering. It is not about post-training quantization. </p>

Reply:
using fewer clusters is less confusing than low-bits in my opinion